lib/galaxy/tool_util/parser/xml.py

```diff
@@ -603,6 +603,20 @@ def parse_creator(self):
             creators.append(creator_as_dict)
         return creators
 
+    def parse_funding(self):
```

Comment:
Maybe add typing information to new functions?

Reply:
Also good point. Also documentation ... my bad.